src/socket/address.cr

```diff
@@ -80,10 +80,10 @@ class Socket
     BROADCAST6   = "ff0X::1"
 
     getter port : Int32
+    getter addr6 : LibC::In6Addr?
```

Comment:
These ivars are implementation details. They're not part of the public interface on purpose.

Reply:
I meant, these getters shouldn't be added at all, neither as documented nor undocumented methods. You can use the ivars directly in your code.

Reply:
How do I do that?

Reply:
Simply revert the changes of this file: remove the getters and readd:
```cr
@addr6 : LibC::In6Addr?
@addr4 : LibC::InAddr?
```

Reply:
wow had no idea you could do that.
Sweet